pkg/server/server_test.go

```diff
@@ -1570,7 +1570,8 @@ func TestDelegateCheckResolver(t *testing.T) {
 	})
 	t.Run("default_check_resolver_alone", func(t *testing.T) {
 		cfg := serverconfig.DefaultConfig()
-		require.False(t, cfg.DispatchThrottling.Enabled)
+		require.False(t, cfg.CheckDispatchThrottling.Enabled)
```

Comment:
Once you re-introduce the `DispatchThrottling` config field back into the server config, you'll want to reintroduce the line above. There should only be additions here, not changes/removals.